packages/adblocker/src/engine/engine.ts

```diff
@@ -908,31 +1023,49 @@ export default class FilterEngine extends EventEmitter<
     }
 
     // Collect all CSP directives and keep track of exceptions
-    const disabledCsp = new Set();
-    const enabledCsp = new Set();
+    const exceptions: Map<string | undefined, NetworkFilter> = new Map();
+    const filters: NetworkFilter[] = [];
```

Comment:
Nit. I think it's best to name these more specifically to hint that they are CSP filters.
```suggestion
    const cspExceptions: Map<string | undefined, NetworkFilter> = new Map();
    const cspFilters: NetworkFilter[] = [];
```